src/platform_impl/web/stdweb/mod.rs

```diff
@@ -31,6 +32,14 @@ impl WindowExtStdweb for Window {
     fn canvas(&self) -> CanvasElement {
         self.window.canvas().raw().clone()
     }
+
+    fn is_dark_mode(&self) -> bool {
+        let is_dark_mode = js! {
+            return (window.matchMedia && window.matchMedia("(prefers-color-scheme: dark)").matches)
+        };
```

Comment:
Might want to throw a TODO here to upstream these APIs to stdweb

Reply:
Done.